lib/addons/p5.dom.js

```diff
@@ -1454,6 +1455,68 @@
     return this;
   };
 
+  /**
+   *
+   * Checks if specified class already set to element
+   *
+   * @method hasClass
+   * @returns {boolean} a boolean value if element has specified class
+   * @param c {String} class name of class to check
+   * @example
+   * <div class='norender'><code>
+   *
+   * var div;
+   *
+   * function setup() {
+   *   div = createDiv('div');
+   *   div.addClass('show');
+   * }
+   *
+   * function mousePressed() {
+   *   if (div.hasClass('show')) {
+   *       div.addClass('show');
+   *   } else {
+   *       div.removeClass('show');
+   *   }
+   * }
+   * </code></div>
+   */
+  p5.Element.prototype.hasClass = function(c) {
+    var regex = new RegExp('(?:[a-zA-Z-_\d:]\s|^)' + c + '(?:\s[a-zA-Z-_\d:]|$)');
```

Comment:
The tests doesn't seem to like this regex, you should be able to run the prettier step to automatically fix this

Reply:
I saw that but I tested my regex on https://regexr.com/ -- it is absolutly fine. But suggested ESlint prettier code does not work on that sandbox. So I will dig into it now.

Reply:
@OleksiiBulba stepping back for a second, why do we need a regex for this? From looking at https://developer.mozilla.org/en-US/docs/Web/API/Element/className, we should be able to parse that into a list of classnames with a simple [String.split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split), and then use [Array.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) to see if the given class is in the list. does that seem reasonable?

Reply:
@outofambit Can we use method **contains** (**add**, **remove**, **toggle** etc.) from https://developer.mozilla.org/en-US/docs/Web/API/Element/classList ?

Reply:
@OleksiiBulba yes! that sounds great :D